### PR TITLE
updating data sources

### PR DESCRIPTION
**Pull request by hayleycd**
Addresses #1391 

Preview [here](https://hayleycd.github.io/osv.dev/data/)

Brings our docs up to date with [osv-schema](https://github.com/ossf/osv-schema#readme)

### CODE REVIEW

File: docs/data.md [hunk 1]
```diff
@@ -40,6 +40,21 @@ The following ecosystems have vulnerabilities encoded in this format:
     ([CC0 1.0](https://github.com/cloudsecurityalliance/gsd-database/blob/main/LICENSE))
 -   [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
     ([CC-BY 4.0](https://github.com/google/oss-fuzz-vulns/blob/main/LICENSE))
+-   [LoopBack Advisory Database](https://github.com/loopbackio/security/tree/main/advisories)
```
Comment (oliverchang) [inline docs/data.md:43]: We aren't ingesting this just yet.

File: docs/data.md [hunk 2]
```diff
@@ -40,6 +40,21 @@ The following ecosystems have vulnerabilities encoded in this format:
...
+    ([MIT](https://github.com/loopbackio/security/blob/main/LICENSE))
+-   [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+    ([BSD](https://rockylinux.org/licensing))
+-   [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+    ([MIT](https://github.com/AlmaLinux/osv-database/blob/master/LICENSE))
+-   [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+    ([CC0 1.0](https://github.com/haskell/security-advisories/blob/main/LICENSE.txt))
+-   [VMWare Photon OS](https://github.com/vmware/photon/wiki/Security-Advisories) (unofficial)
```
Comment (oliverchang) [inline docs/data.md:51]: We aren't ingesting this yet.

File: docs/data.md [hunk 3]
```diff
@@ -40,6 +40,21 @@ The following ecosystems have vulnerabilities encoded in this format:
...
+-   [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
```
Comment (oliverchang) [inline docs/data.md:52]: We aren't ingesting this yet.

File: docs/data.md [hunk 4]
```diff
@@ -40,6 +40,20 @@ The following ecosystems have vulnerabilities encoded in this format:
...
 -   [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
     ([CC-BY 4.0](https://github.com/google/oss-fuzz-vulns/blob/main/LICENSE))
+    ([MIT](https://github.com/loopbackio/security/blob/main/LICENSE))
+-   [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+    ([BSD](https://rockylinux.org/licensing))
+-   [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+    ([MIT](https://github.com/AlmaLinux/osv-database/blob/master/LICENSE))
+-   [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+    ([CC0 1.0](https://github.com/haskell/security-advisories/blob/main/LICENSE.txt))
+-   [VMWare Photon OS](https://github.com/vmware/photon/wiki/Security-Advisories) (unofficial)
```
Comment (hayleycd) [inline docs/data.md:50]:
```suggestion
```

File: docs/data.md [hunk 5]
```diff
@@ -40,6 +40,19 @@ The following ecosystems have vulnerabilities encoded in this format:
...
 -   [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
     ([CC-BY 4.0](https://github.com/google/oss-fuzz-vulns/blob/main/LICENSE))
+    ([MIT](https://github.com/loopbackio/security/blob/main/LICENSE))
+-   [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+    ([BSD](https://rockylinux.org/licensing))
+-   [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+    ([MIT](https://github.com/AlmaLinux/osv-database/blob/master/LICENSE))
+-   [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+    ([CC0 1.0](https://github.com/haskell/security-advisories/blob/main/LICENSE.txt))
+-   [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
```
Comment (hayleycd) [inline docs/data.md:50]:
```suggestion
```

**Comment by hayleycd** [comment]
Thanks @oliverchang ! I have removed the sources that aren't in our database yet.

File: docs/data.md [hunk 6]
```diff
@@ -40,6 +40,18 @@ The following ecosystems have vulnerabilities encoded in this format:
...
+    ([MIT](https://github.com/AlmaLinux/osv-database/blob/master/LICENSE))
+-   [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+    ([CC0 1.0](https://github.com/haskell/security-advisories/blob/main/LICENSE.txt))
+    ([Apache 2.0](https://github.com/bitnami/vulndb/blob/main/LICENSE.md))
+-   [RConsortium Advisory Database](https://github.com/RConsortium/r-advisory-database)
+    ([Apache 2.0](https://github.com/RConsortium/r-advisory-database/blob/main/LICENSE))
+-   [Python Software Foundation Database](https://github.com/psf/advisory-database)
+    ([CC-BY 4.0](https://github.com/psf/advisory-database/blob/main/LICENSE))
 
 ## Converted data
```
Comment (andrewpollock) [inline docs/data.md:56]: I think we've blurred reality a little bit in this section. We're not maintaining conversion pipelines for:

- AlmaLinux
- Bitnami I'm confused about completely
- crates.io
- GitHub Actions
- Go I'm unclear on the accuracy of this statement
- Haskell
- Hex
- Linux Kernel
- Maven
- npm
- NuGet
- Packagist
- Photon OS
- Pub
- Python
- R (CRAN and Bioconductor)

These all either come via GitHub Security Advisories or native maintained record sources

File: docs/data.md [hunk 7]
```diff
@@ -40,6 +40,18 @@ The following ecosystems have vulnerabilities encoded in this format:
...
 -   [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
     ([CC-BY 4.0](https://github.com/google/oss-fuzz-vulns/blob/main/LICENSE))
+    ([MIT](https://github.com/loopbackio/security/blob/main/LICENSE))
+-   [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+    ([BSD](https://rockylinux.org/licensing))
+-   [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+    ([MIT](https://github.com/AlmaLinux/osv-database/blob/master/LICENSE))
+-   [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+    ([CC0 1.0](https://github.com/haskell/security-advisories/blob/main/LICENSE.txt))
+    ([Apache 2.0](https://github.com/bitnami/vulndb/blob/main/LICENSE.md))
```
Comment (hayleycd) [inline docs/data.md:50]:
```suggestion
```

File: docs/data.md [hunk 8]
```diff
@@ -40,6 +40,18 @@ The following ecosystems have vulnerabilities encoded in this format:
     ([CC0 1.0](https://github.com/cloudsecurityalliance/gsd-database/blob/main/LICENSE))
 -   [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
     ([CC-BY 4.0](https://github.com/google/oss-fuzz-vulns/blob/main/LICENSE))
+    ([MIT](https://github.com/loopbackio/security/blob/main/LICENSE))
```
Comment (hayleycd) [inline docs/data.md:43]:
```suggestion
```

File: docs/data.md (hunk 6 above)
Comment (hayleycd) [reply docs/data.md:54]: So I am a little confused. I think you didn't look at the changes in context. There is a whole section between the conversion pipeline section and these changes. These are just the covered ecosystems.

File: docs/data.md [hunk 9]
```diff
@@ -57,19 +67,24 @@ Between the data served in OSV and the data converted to OSV the following ecosy
 -   AlmaLinux
 -   Alpine
 -   Android
+-   Bitnami
```
Comment (hayleycd) [inline docs/data.md:70]:
```suggestion
```

File: docs/data.md [hunk 10]
```diff
@@ -61,15 +71,19 @@ Between the data served in OSV and the data converted to OSV the following ecosy
...
 -   Go
+-   Haskell
 -   Hex
 -   Linux kernel
 -   Maven
 -   npm
 -   NuGet
 -   OSS-Fuzz
 -   Packagist
+-   Photon OS
```
Comment (hayleycd) [inline docs/data.md:82]:
```suggestion
```

File: docs/data.md (hunk 6 above)
Comment (hayleycd) [reply docs/data.md:54]: @andrewpollock See the [preview here](https://hayleycd.github.io/osv.dev/data/#converted-data) and let me know if it looks ok in the context of the whole page.

Comment (andrewpollock) [reply docs/data.md:54]: My apologies!

**Review by andrewpollock — APPROVED** [review]

**Review by oliverchang — APPROVED** [review]
thanks!

**Pull request merged**